wp-content/themes/heideandcook/views/base.html.twig

```diff
@@ -5,6 +5,11 @@
         <meta name="viewport" content="height=device-height,width=device-width,initial-scale=1">
         <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1"/>
         <title>{{ wp_get_the_title() }}</title>
+        <link rel="stylesheet" type="text/css" href="../src/slick/slick.css"/>
```

Comment:
why not use `{{ head|raw }}`? If this template isn't being used you can delete it.